Dockerfile

```diff
@@ -1,10 +1,24 @@
-FROM python:3.5-alpine
+FROM golang:1.8.0-alpine
+LABEL maintainer "Infinity Works"
 
-RUN pip install prometheus_client requests
+EXPOSE 9171
 
-ENV BIND_PORT 9171
+ENV GOPATH=/go
+ENV LISTEN_PORT=9171
```

Comment:
Given that go-common uses safe defaults, should we just remove this from the dockerfile?